handel/src/aggregation.rs

```diff
@@ -395,9 +400,23 @@ where
         self.current_verification = Some(fut);
         None
     }
+
+    /// Responds to an update received from `node_id` with our own best aggregate for `level_id`.
+    /// If we already have a final result, we send that result instead.
+    fn respond_to_update(&mut self, node_id: usize, level_id: usize) {
```

Comment:
```suggestion
    fn respond_to_update(&mut self, level_id: usize, node_id: usize) {
```
Other functions take `level_id` first and `node_id` second. Careful, also needs updating of the call sites!

Reply:
Only one call side. Changed it there, too.